include/pmacc/eventSystem/Manager.tpp

```diff
@@ -163,6 +163,15 @@ namespace pmacc
         }
     }
 
+    template<typename T_Functor>
+    void Manager::waitFor(T_Functor&& func)
+    {
+        while(!func())
+        {
+            this->execute();
+        }
+    }
+
     inline void Manager::waitForAllTasks()
```

Comment:
This is not really part of this PR, just a note from my previous attempt to make multiple object files. Such methods, implemented outside of the class, should be made inline at declaration (so inside the class), but not here at definition.

Reply:
Ahh yes I can make it `inline`